blueprints/initializer-test/files/tests/unit/initializers/__name__-test.js

```diff
@@ -16,7 +15,7 @@ module('<%= friendlyTestName %>', {
 
 // Replace this with your real tests.
 test('it works', function(assert) {
-  initialize(registry, application);
+  <%= classifiedModuleName %>Initializer.initialize(application);
```

Comment:
The application is only passed as the first argument in Ember 2.1, but master is tracking 2.0.